<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget_MainTool</class>
 <widget class="QTabWidget" name="TabWidget_MainTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab2">
   <property name="maximumSize">
    <size>
     <width>396</width>
     <height>52</height>
    </size>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <attribute name="title">
    <string>Учет</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutDoc"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab1">
   <property name="maximumSize">
    <size>
     <width>396</width>
     <height>47</height>
    </size>
   </property>
   <attribute name="title">
    <string>Таблицы</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutList">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Программа</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSys"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
